src/9.negotiation.js

```diff
@@ -20,6 +20,20 @@ import {
 
 const expUnicodeExSeq = /-u(?:-[0-9a-z]{2,8})+/gi; // See `extension` below
 
+defineProperty(Intl, 'getCanonicalLocales', {
```

Comment:
src/intl.js only defined Intl object. All `defineProperty` are places in respective files (like numberformat or datetimeformat).

Is it ok if I just do what those two files do - import Intl from src/intl.js?